src/applications/burial-poc-v6/pages/ReviewPage.jsx

```diff
@@ -0,0 +1,505 @@
+            value: state?.values?.govtContributions,
+          },
+          amountGovtContribution: {
+            label: 'Amount of government or employer contribution',
+            value: state?.values?.amountGovtContribution,
+          },
+        },
+      },
+      {
+        title: 'Claimant Contact Information',
+        id: 'claimant-contact-information',
+        pageUrl: '/claimant-contact-information',
+        isShown: true,
+        fields: {
+          claimantAddress: {
+            label: 'Claimant Address',
+            value: {
+              country: {
+                label: 'Country',
+                value: state?.values?.claimantAddress?.country,
+              },
+              street: {
+                label: 'Street',
+                value: state?.values?.claimantAddress?.street,
+              },
+              street2: {
+                label: 'Street 2',
+                value: state?.values?.claimantAddress?.street2,
+              },
+              city: {
+                label: 'City',
+                value: state?.values?.claimantAddress?.city,
+              },
+              state: {
+                label: 'State',
+                value: state?.values?.claimantAddress?.state,
+              },
+              postalCode: {
+                label: 'Postal code',
+                value: state?.values?.claimantAddress?.postalCode,
+              },
+            },
+          },
+          claimantEmail: {
+            label: 'Claimant Email',
+            value: state?.values?.claimantEmail,
+          },
+          claimantPhone: {
+            label: 'Claimant Phone Number',
+            value: state?.values?.claimantPhone,
+          },
+        },
+      },
+      // {
+      //   title: "Additional information",
+      //   id: 'additional-information',
+      //   pageUrl: '/additional-information',
+      //   fields: {
+      //     "transportationReceipts": {
+      //       label: "Transportation Receipts?",
+      //       value: state?.values?.transportationReceipts,
+      //     }
+      //   },
+      // }
+    ],
+  };
+
+  return (
+    <Page {...props} nextButtonCustomText="Submit">
+      <article>
+        <h1>{props.title}</h1>
+        <VaOnThisPage />
+
+        {pageData.pages.map(page => {
+          if (page.isShown) {
+            return (
+              <section
+                id={page.id}
+                key={page.id}
+                className="review-page--page-info"
+              >
+                <div className="review-page--page-heading vads-u-justify-content--space-between vads-l-row vads-u-border-bottom--1px vads-u-border-color--link-default">
+                  <h2
+                    id={page.id}
+                    className="vads-u-font-size--h3 vads-u-flex--1 review-page--page-heading--text"
+                  >
+                    {page.title}
+                  </h2>
+                  <Link
```

Comment:
Does this Link component accept an aria-label attribute? I don't think it would clear to screen reader users as to what they are editing.

Reply:
Added the aria-label